src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1701,7 +1146,7 @@ private static void SetNoClientCertificate(SafeWinHttpHandle requestHandle)
                 0);
```

Comment:
We currently don't have that value in `Common/src/Interop/Windows/winhttp/Interop.winhttp_types.cs`

There are other "0" sentinel values as well (for things like *NO_PROXY and I chose at that time not to list those 0 / IntPtr.Zero things in that winhttp_types.cs file.  Instead, I though it was cleaner to use pass in 0.

I'll add a TODO to consider this and add WINHTTP_NO_CLIENT_CERT_CONTEXT and the other sentinels as well.